src/sys/mman.rs

```diff
@@ -184,6 +184,40 @@ libc_bitflags! {
         #[cfg(target_os = "openbsd")]
         #[cfg_attr(docsrs, doc(cfg(all())))]
         MAP_CONCEAL;
+        /// Pages aligned on 64kb
+        #[cfg(target_os = "netbsd")]
+        #[cfg_attr(docsrs, doc(cfg(all())))]
```

Comment:
Please rebase and remove the `#[cfg_attr(docsrs, doc(cfg(all())))]` attributes from all the variants as we have removed them all from the codebase (#2192)